game/src/constants.js

```diff
@@ -6,5 +6,11 @@ const greetings = [
     'It\'s a plane!',
     '1000% more Cube!',
     'A Whole New Dimension!',
+    'Explosive pleasure!',
```

Comment:
I'm not sure this line fits with the others. Do you mind removing or rewording it?

Reply:
Okay will do